README.md

```diff
@@ -8,21 +8,214 @@
+
+5. Store the access token securely for use in your application.
 
 ## Quickstart
 
-[//]: # (TODO: Add a quickstart guide to demonstrate a basic functionality of the module, including sample code snippets.)
+To use the `HubSpot CRM Leads Connector` in your Ballerina application, update the `.bal` file as follows:
+
+### Step 1: Import the module
+
+Import the `hubspot.crm.objects.leads` module and `oauth2` module.
+
+```ballerina
+import ballerinax/hubspot.crm.objects.leads as hsleads;
+import ballerina/oauth2;
+```
+
+### Step 2: Instantiate a new connector
+
+1. Create a `Config.toml` file and, configure the obtained credentials in the above steps as follows:
+
+   ```toml
+    clientId = "<Client Id>"
+    clientSecret = "<Client Secret>"
+    refreshToken = "<Refresh Token>"
+   ```
+
+2. Instantiate a `hsleads:ConnectionConfig` with the obtained credentials and initialize the connector with it.
+
+    ```ballerina 
+    configurable string clientId = ?;
+    configurable string clientSecret = ?;
+    configurable string refreshToken = ?;
+
+    final hsleads:ConnectionConfig hsleadsConfig = {
+        auth : {
+            clientId,
+            clientSecret,
+            refreshToken,
+            credentialBearer: oauth2:POST_BODY_BEARER
+        }
+    };
+
+    final hsleads:Client hsleads = check new (hsleadsConfig);
+    ```
+
+### Step 3: Invoke the connector operation
+
+Now, utilize the available connector operations. A sample usecase is shown below.
+
+#### Create a Lead
+    
+```ballerina
+public function main() returns error? {
+   SimplePublicObjectInputForCreate payload = {
+      {
+         "associations": [
+            {
+               "types": [
+                  {
+                     "associationCategory": "HUBSPOT_DEFINED",
+                     "associationTypeId":578
+                  }
+               ],
+               "to": {
+                  "id": "YOUR_CONTACT_ID"
+               }
+            }
+         ],
+         "properties": {
+            "hs_lead_name": "John Doe"
+         }
+      }
+   }
+   hsleads:SimplePublicObject createLead = check hsleads->/.post(payload);
+}
+```
+
+### Step 4: Run the Ballerina application
+
+```bash
+bal run
+```
 
 ## Examples
 
-The `Ballerina HubSpot CRM Leads Connector` connector provides practical examples illustrating usage in various scenarios. Explore these [examples](https://github.com/module-ballerinax-hubspot.crm.object.leads/tree/main/examples/), covering the following use cases:
+The `Ballerina HubSpot CRM Leads Connector` connector provides practical examples illustrating usage in various scenarios. Explore these [examples](https://github.com/ballerina-platform/module-ballerinax-hubspot.crm.object.leads/tree/38bfdb49b5b8d209729700d8e52961f6a08d632f/examples), covering the following use cases:
 
-[//]: # (TODO: Add examples)
+- [Real Estate Inquiry Leads](examples/real_estate_inquiry_leads)
+- [Fitness Center Leads](examples/fitness_center_leads)
```

Comment:
Can you restructure the examples by following this structure ?

https://github.com/ballerina-platform/module-ballerinax-hubspot.crm.object.deals/blob/d9c9c839279da32f429477537ccd250b399f02af/README.md#examples